pipelines/broad/internal/rna_seq/BroadInternalRNAWithUMIs.wdl

```diff
@@ -7,7 +7,7 @@ import "../../../../tasks/broad/Utilities.wdl" as utils
 
 workflow BroadInternalRNAWithUMIs {
 
-  String pipeline_version = "1.0.2"
+  String pipeline_version = "1.0.3"
```

Comment:
Back to 1.0.1 (see my comment on the changelog).

Reply:
👍 thanks!